client/packages/system/src/Stock/ListView/ListView.tsx

```diff
@@ -233,3 +249,39 @@ export const StockListView: FC = () => (
     <StockListComponent />
   </TableProvider>
 );
+
+const UnitsAndMaybeDosesCell = (
+  props: CellProps<StockLineRowFragment> & { displayDoses?: boolean }
+) => {
+  const { rowData, column } = props;
+  const t = useTranslation();
+  const units = Number(column.accessor({ rowData })) ?? 0;
+  const { isVaccine, dosesPerUnit } = rowData.item;
+
+  // Doses should always be a whole number, round if fractional packs are giving us funky decimals
+  const doseCount = NumUtils.round(dosesPerUnit * units);
+  return (
+    <Box
+      sx={{
+        display: 'flex',
+        alignItems: 'center',
+        justifyContent: 'flex-end',
+        width: '100%',
+      }}
+    >
+      <NumberCell {...props} />
+      {props.displayDoses && isVaccine && (
+        <Typography
+          sx={{
+            fontSize: 'small',
+            fontStyle: 'italic',
```

Comment:
just personal preference but the italics looks so off here... would just keep it normal..